virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -11,6 +11,7 @@ You can also import virtual machines with existing workloads that depend on acce
 
 To attach a virtual machine to an additional network:
 
+. Configure a Linux bridge.
```

Comment:
Moving my previous comment here:
How to configure a bridge?
I think that a link to the page that explains that (https://deploy-preview-43988--osdocs.netlify.app/openshift-enterprise/latest/virt/node_network/virt-updating-node-network-config.html) may be useful.

Reply:
@yossisegev You are right that the link is useful, which is why it's in the "Creating a Linux bridge using a node network configuration policy" section immediately following the opening section.

A big part of the bug is that we want to emphasize users should use NADs instead of NNCPs when creating VLANs. But we also still want to tell them how to create VLANs using NNCPs.

So I put in a note in the opening section saying NADs are better, but I also included the link you suggest in the NNCP section so the user can still find that additional information relatively easily. .